client/signup/accordion-form/accordion-form.tsx

```diff
@@ -3,20 +3,22 @@ import { ChangeEvent, useState } from 'react';
 import AccordionFormSection from './accordion-form-section';
 import {
 	AccordionSectionProps,
-	sectionGeneratorReturnType,
+	SectionGeneratorReturnType,
 	ValidationErrors,
 	ValidatorFunction,
 } from './types';
 
 interface AccordionFormProps< T > {
-	sectionGenerator: () => (
-		props: sectionGeneratorReturnType< T >
+	generatedSections?: any[];
```

Comment:
In the view of future refactoring added this prop that lives inside react's render loop.